test/resolveGuildTemplateCode.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/* eslint-env jest */
+
+const { DataResolver } = require('../src');
+
+describe('resolveGuildTemplateCode', () => {
+  test('basic', () => {
+    expect(DataResolver.resolveGuildTemplateCode('https://discord.new/abc')).toBe('abc');
```

Comment:
I wasn't sure if this test is really necessary, but because I technically changed the code in `resolveGuildTemplateCode`, I added this just in case. I'm happy to remove it if this is unnecessary.

There don't seem to be many existing tests, so I just copied what was in `test/escapeMarkdown.test.js`.